modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -415,7 +415,10 @@ func (suite *TendermintTestSuite) TestUpdateState() {
 				suite.Require().True(path.EndpointA.GetClientState().GetLatestHeight().LT(clientMessage.GetHeight()))
 			},
 			func() {
-				suite.Require().True(path.EndpointA.GetClientState().GetLatestHeight().LT(updatedClientState.GetLatestHeight())) // new update, updated client state should have changed
+				bz := clientStore.Get(host.ClientStateKey())
+				updatedClientState := clienttypes.MustUnmarshalClientState(suite.chainA.App.AppCodec(), bz)
+
+				suite.Require().True(path.EndpointA.GetClientState().GetLatestHeight().EQ(updatedClientState.GetLatestHeight())) // new update, updated client state should have changed
```

Comment:
`path.EndpointA.GetClientState()` will now return latest client state as `UpdateState` sets it in store, so here we test for equality rather than less than

Reply:
maybe we should use `clientMessage.GetHeight()` instead of the `updatedClientState`?

Reply:
Nice yea, I agree, cleaner :)

Updated!